### ISSUE TITLE
Updating Hit Category severity doesn't update the flight hits

### ISSUE THREAD

**Issue by gtasautumn** [US-CBP/GTAS-UI#539]
**Describe the bug**
I generated a flight with manual hits and watchlist hits and then decided to change the severity on one of them down to yellow/low/normal. On the FlightPax page, the the severity was updated, but on the Flights All tab, it remained red/high.

**To Reproduce**
1. Go to Flights and find a flight with hits on the All tab. Note the hit counts.
2. Find a passenger on that flight with a hit and note the hit category, passenger
3. Go to Admin > Hit Categories
4. Edit the severity for that category
5. Go back to FlightPax and verify that the passenger's hit associated with that category gets updated.
6. go back to Flights and verify that the counts by severity do not change.

**Expected behavior**
This is a question. I am actually not sure if it should update at all, perhaps we need to discuss. But whatever the desired behavior is, the hit counts should be in sync between Flights and FlightPax.

**Comment by dje04001** [comment]
I don't think this should update the severity of prior hits. At the time of the hit, the assessed severity attached to that category was X. We shouldn't retroactively change X. I think the severity of certain threats will change over time, but record should be maintained that at one point, that category was a certain level.

This change in hit category severity should be reflected in all of the hits created after this change is saved.

**Comment by tjwdev** [comment]
Per Meeting: 

Potential solution is to archive existing hit categories when being edited, this maintains the existing hit relationship for all EXISTING hits. Then create a new hit category with that edited value for all.

Lingering question: What happens when we edit categories being used currently in rules. This would maintain those rules as the archived category instead of the new one.
